bh/src/assets/scss/forms.scss

```diff
@@ -4,7 +4,7 @@
 
 
 @mixin form-component-sizes() {
-  height: $form-component-height-normal;
+  height: $form-component-height-normal + 1rem;
```

Comment:
Don't do this.. the variable `$form-component-height-normal` no make sense if you add 1rem.